src/test/a11y/a11y.ts

```diff
@@ -93,6 +93,8 @@ function testAccessibility(url: string): void {
 
 describe('Accessibility',  () => {
   readRoutes().forEach(route => {
-    testAccessibility(route);
+    if (route != '/courtlistall' && route != '/courtlist/:input' && route != '/hearings/:courtId') {
```

Comment:
if we add api prefix to the route as I mentioned above we can just ignore all routes on /api/* path rather than having to define each